overops_collector/app/takipi/VERSION

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
The entire /takipi/ folder should not be in the repo. Only include the compressed tarball (.tar.gz file).